deployment/aws-provider-installer.yaml

```diff
@@ -62,6 +62,8 @@ spec:
           imagePullPolicy: Always
           args:
               - --provider-volume=/etc/kubernetes/secrets-store-csi-providers
+              - --qps=5
```

Comment:
Same here, let's leave these out of the default YAML file. Customers can refer to the README for guidance on how to edit it to adjust these.

Reply:
Will do.